raw/patterns/home.php

```diff
@@ -0,0 +1,165 @@
+<figure class="wp-block-image aligncenter size-full is-resized"><a href="https://raw.mystagingwebsite.com/page/"><img src="<?php echo esc_url( get_stylesheet_directory_uri() ); ?>/assets/images/btn_plus.png" alt="" class="wp-image-161" width="64" height="64"/></a></figure>
+<!-- /wp:image --></div>
+<!-- /wp:group -->
+
+<!-- wp:spacer {"height":"8rem","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+<div style="margin-top:0px;margin-bottom:0px;height:8rem" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:spacer {"height":"4rem","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+<div style="margin-top:0px;margin-bottom:0px;height:4rem" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column {"style":{"border":{"color":null,"style":null,"width":null,"top":{"style":"solid","width":"8px"},"right":{"style":"none","width":"0px"},"bottom":{"style":"none","width":"0px"},"left":{"style":"none","width":"0px"}},"spacing":{"padding":{"top":"38px"}}}} -->
+<div class="wp-block-column" style="border-top-style:solid;border-top-width:8px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-style:none;border-left-width:0px;padding-top:38px"><!-- wp:heading {"level":1,"align":"wide"} -->
+<h1 class="wp-block-heading alignwide">Six most appreciated Brutalist buildings</h1>
+<!-- /wp:heading --></div>
+<!-- /wp:column -->
+
+<!-- wp:column {"style":{"border":{"color":null,"style":null,"width":null,"top":{"width":"2px","style":"solid"},"right":{"width":"0px","style":"none"},"bottom":{"width":"0px","style":"none"},"left":{"width":"0px","style":"none"}},"spacing":{"padding":{"top":"44px"}}}} -->
+<div class="wp-block-column" style="border-top-style:solid;border-top-width:2px;border-right-style:none;border-right-width:0px;border-bottom-style:none;border-bottom-width:0px;border-left-style:none;border-left-width:0px;padding-top:44px"><!-- wp:paragraph {"fontSize":"medium"} -->
+<p class="has-medium-font-size">Brutalism was also influenced by the work of the Bauhaus school, which emphasized functionalism and simplicity in design.</p>
+<!-- /wp:paragraph --></div>
+<!-- /wp:column --></div>
+<!-- /wp:columns -->
+
+<!-- wp:spacer {"height":"4rem","style":{"spacing":{"margin":{"top":"0px","bottom":"0px"}}}} -->
+<div style="margin-top:0px;margin-bottom:0px;height:4rem" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"align":"wide"} -->
+<div class="wp-block-columns alignwide"><!-- wp:column -->
+<div class="wp-block-column"><!-- wp:block {"ref":307} /--></div>
```

Comment:
<img width="604" alt="image" src="https://github.com/Automattic/themes/assets/146530/a2b7f475-ac1b-45e4-abd4-e6ee3905eb87">
This is what I see from the editor.

Reply:
Confirmed that was the case.

It will be grand for CBT to export these (or even better, for Gutenberg to manage this for themes).  

Until then I just copied them out to theme patterns.